hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationRunnerImpl.java

```diff
@@ -196,8 +196,9 @@ private void checkNodeState(Operation op) {
             return;
         }
 
+        MemberImpl localMember = node.getLocalMember();
         if (state == NodeState.SHUT_DOWN) {
-            throw new HazelcastInstanceNotActiveException("This node is shut down! Operation: " + op);
+            throw new HazelcastInstanceNotActiveException(localMember + " is shut down! Operation: " + op);
```

Comment:
doesn't this message still contain `this`? See toString [here](https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/instance/AbstractMember.java#L186)